client/src/components/EventCalendar/index.tsx

```diff
@@ -0,0 +1,152 @@
+import { FC, useState, useEffect } from 'react'
+import FullCalendar, { formatDate } from '@fullcalendar/react'
+import dayGridPlugin from '@fullcalendar/daygrid'
+import timeGridPlugin from '@fullcalendar/timegrid'
+import interactionPlugin from '@fullcalendar/interaction'
+import './style.css'
+import { useNavigate } from 'react-router-dom'
+import ApiService from '../../services/ApiService'
+
+const EventCalendar:FC = () => {
+  const [initialEvents, setInitialEvents] = useState([])
+  const [interestedEvents, setInterestedEvents] = useState([])
+  const navigate = useNavigate()
+  const handleEventClick = (clickInfo:any):void => {
+    const eventId = clickInfo?.event?.extendedProps.eventId
+
+    if (window.confirm(`Are you sure you want to go to the event '${clickInfo.event.title}'`)) {
+      navigate(`/event/${eventId}`)
+    }
+  }
```

Comment:
don't use `window.confirm` you can use a component from MUI
or you can use SweetAlert

or you don't have to confirm the user when he clicks on the event (and that's my opinion)